### PR TITLE
Clean up further mentions of "beta" in the docs

### PR DESCRIPTION
**Pull request by nmuesch**
### What does this PR do?

Remove mentions of beta in the github docs

### Motivation

Clean up documentation

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

File: Dockerfiles/dogstatsd/socat-proxy/README.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ spec:
       containers:
         - **[insert producer container specs]**
         - name: socat
-          image: datadog/dogstatsd-socat-proxy:beta
+          image: datadog/dogstatsd-socat-proxy:latest
```
Comment (arbll) [inline Dockerfiles/dogstatsd/socat-proxy/README.md]: It seems there is no `latest` tag for dogstatsd-socat-proxy : https://hub.docker.com/r/datadog/dogstatsd-socat-proxy/tags/

File: pkg/config/config_template.yaml [hunk 2]
```diff
@@ -147,6 +147,13 @@ api_key:
 # The port for the go_expvar server
 # dogstatsd_stats_port: 5000
 #
```
Comment (arbll) [inline pkg/config/config_template.yaml]: Good job noticing this was missing from the template 👍

Comment (nmuesch) [reply pkg/config/config_template.yaml]: Thanks :)

File: Dockerfiles/dogstatsd/socat-proxy/README.md (hunk 1 above)
Comment (nmuesch) [reply Dockerfiles/dogstatsd/socat-proxy/README.md]: Good catch. I changed that back to beta for now.

**Review by ChristineTChen — DISMISSED** [review]
Looks good to me!!

File: docs/agent/upgrade.md [hunk 3]
```diff
@@ -1,5 +1,7 @@
 # Upgrade to Agent 6
 
+** Important ** Datadog Agent 6.0.0 has officially been released. Please utilize the in application [install instructions](https://app.datadoghq.com/account/settings#agent) for our pre packaged binaries.
```
Comment (derekwbrown) [inline docs/agent/upgrade.md]: Little nit, but I think it should be "in-application" (hyphen)

**Review by derekwbrown — DISMISSED** [review]
LGTM except for one nit; can push after that

File: docs/agent/upgrade.md (hunk 3 above)
Comment (nmuesch) [reply docs/agent/upgrade.md]: Sounds good, will change and merge. Thanks!

**Review by arbll — APPROVED** [review]

**Review by mfpierre — APPROVED** [review]
:shipit:

**Comment by olivielpeau** [comment]
Merging since we are GA now!

@DataDog/kenafeh can you review and change anything that doesn't look correct?

**Pull request merged**